docs/docs/modifying-a-starter.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Modifying a Starter
+---
+
+While a Gatsby [starter](/docs/starters/) is a working website out of the box, chances are you'll want to modify your starter at some point. Building on top of a starter reduces the amount of boilerplate code you need to write yourself. You're therefore able to write your own content, functionality, and styling faster.
+
+## Prerequisites
+
+What you need to know will depend on the starter you choose and the data or functionality you'd like to modify. Even if you choose _not_ to modify the starter's components, you may still want to update text, use data from an external source, and modify the style (CSS) of the site. To do this, you'll write some [Markdown](/docs/mdx/markdown-syntax/) and [JSON](https://www.digitalocean.com/community/tutorials/an-introduction-to-json).
+
+To start modifying the functionality of a starter, you'll want a basic understanding of [JSX](https://reactjs.org/docs/introducing-jsx.html) syntax for updating components and making new ones. You'll also want some knowledge of [GraphQL](https://graphql.org/learn/) for querying your data. Start with these and add to your skills as you continue to add functionality to your starter.
+
+## Choosing a starter
+
+To choose a starter, first consider the functionality you need. Are you building an ecommerce site? A blog? Do you already know what data sources you'll want to use? Find a starter that fulfills your requirements by using the searchbar and filters in the [Gatsby Starter Library](/starters/).
+
+If you're not sure what to choose or want only the most essential functionality, try customizing either [`gatsby-starter-blog`](https://github.com/gatsbyjs/gatsby-starter-blog) (if you're primarily using this site as a blog) or [`gatsby-starter-default`](https://github.com/gatsbyjs/gatsby-starter-default). These official starters are maintained by Gatsby and are great options, particularly for your first Gatsby site.
+
+## Exploring the folder structure
+
+Taking the default Gatsby starter as an example, dedicate some time to looking through some of the most important files in the repository (repo). You'll find most of these in the root (or _main_) folder of your project. Doing so will give you a clearer picture of how the various parts of your Gatsby site work together.
+
+![Default starter file structure in VS Code](./images/default-starter-file-structure.png)
```

Comment:
Yes, totally agreed on the size issue. I went with showing more of the editor but blurred the rest of it out. I think this will give just a little more context and still keep the focus on the structure.